### ISSUE TITLE
Interactive tutorial is not accessible to blind programmers

### ISSUE THREAD

**Issue by dlangBugzillaToGithub** [dlang/dlang.org#4069]
### CC List

* berni44 (@WalterBright)

### Description

````markdown
jareds was kind enough to do a quick evaluation of dlang.org's accessibility to blind programmers. Here is his report:

"The site appears to generally be accessible with the exception of the interactive tutorial. I'm not a web accessibility expert so don't know the details. There are two major issues. The biggest one is my inability to arrow through the code in an edit field. I can read the text outside of the edit field, but it's difficult since line numbers are interspersed with the text. My assumption is that your using an open source editor which is inaccessible, but I have not had the time to look at the tutorial on Github. The second issue is that there are buttons that my screen reader reads as "button". While run, edit, and format are read properly several other buttons are not."
````

**Comment by dlangBugzillaToGithub** [comment]
#### bugzilla (@WalterBright) commented on 2019-12-18T10:36:19Z



````markdown
Just go to one of that pages (e.g. https://tour.dlang.org/tour/en/welcome/welcome-to-d) and remove the stylesheet (i.e. choose page style = "no style" or something similar in your browser) to see, what is meant.
````
